packaging/hip-targets-release.cmake

```diff
@@ -17,12 +17,17 @@ set_target_properties(hip::hip_hcc_static PROPERTIES
   IMPORTED_LINK_INTERFACE_LANGUAGES_RELEASE "CXX"
   IMPORTED_LOCATION_RELEASE "${_IMPORT_PREFIX}/lib/libhip_hcc_static.a"
   )
-else()
+elseif(HIP_COMPILER STREQUAL "hcc")
 set_target_properties(hip::hip_hcc_static PROPERTIES
   IMPORTED_LINK_INTERFACE_LANGUAGES_RELEASE "CXX"
   IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE "hc_am"
```

Comment:
This should be removed as well. It should be the same between hip-clang and hcc.